src/en/readkomik/src/eu/kanade/tachiyomi/extension/en/readkomik/ReadKomik.kt

```diff
@@ -1,14 +1,128 @@
 package eu.kanade.tachiyomi.extension.en.readkomik
 
 import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.GET
 import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import okhttp3.HttpUrl.Companion.toHttpUrl
 import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Element
 
-class ReadKomik : MangaThemesia("Readkomik", "https://readkomik.com", "en") {
+class ReadKomik : MangaThemesia(
+    "Readkomik",
+    "https://rkreader.org",
+    "en",
+    "/archives/manga",
+) {
+    override val versionId = 2
```

Comment:
why bump this? I think `/manga/...` urls are redirecting to `/archives/manga/...` tho; 
you could add extra logic to details and chapters request to prevent this redirect

Reply:
Resolved: [a962f64](https://github.com/keiyoushi/extensions-source/pull/2683/commits/a962f6449fadd5a2d77b7a89aceb75d82903f7cb)